### DISCUSSION TITLE
Issue with pyro3d_wood_char.fds

### DISCUSSION THREAD

**Discussion by rmcdermo** [firemodels/fds#12664] [General]
Continuation of [this](https://groups.google.com/g/fds-smv/c/q3cIKjjtv3E) Google group discussion.

**Comment by zishanulh** [comment]
Is there a way to perform 3D pyrolysis in the current version of FDS?

**Comment by mcgratta** [comment]
Yes. The parameters and methodology are same for 1D or 3D. The 3D solver transports heat in all three directions, but the reaction source term is handled the same way as before. Note that there is no solid phase mass transport in either 1D or 3D.

**Comment by mcgratta** [comment]
There are some examples of 3D heat transfer and pyrolysis in `Samples/Heat_Transfer`

**Reply by zishanulh** [reply]
Thank you so much for your response. I appreciate it.
I was able to tweak 'pyro3d_wood_char.fds' and run it, although some diagnostics like 'SOLID CELL DENSITY' did not work. I will also look into the Heat_Transfer folder.

**Comment by mcgratta** [comment]
Where do you see `SOLID CELL DENSITY`? Make sure you are running the latest code (6.9.0) and using the latest documentation and sample cases.

**Reply by zishanulh** [reply]
`SOLID CELL DENSITY` was in the pyro3d_wood_char.fds available in the 6.8.0 example library. And yes, I need to upgrade to 6.9.0, thanks for your suggestion.
